src/transformState.js

```diff
@@ -6,6 +6,33 @@
  */
 function transformState(state, actions) {
   // write code here
+  for (const action of actions) {
+    const { type } = action;
```

Comment:
You could destructure all needed variables at once
```suggestion
     const { type, extraData, keysToRemove } = action;
```